src/api/Auth.service.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable */
+import "whatwg-fetch";
+import AsyncStorage from "@react-native-community/async-storage";
+import { AUTH_LOGIN_ENDPOINT, AUTH_REGISTRATION_ENDPOINT, RESET_PASSWORD_ENDPOINT } from "config";
+import { BaseService } from "api";
+/* eslint-enable */
+
+export default class AuthService {
```

Comment:
maybe it make sense just to use inheritance, so AuthService would extend BaseService, but it's might be subjective 🤷‍♂️